docs/varnish/vcl/varnish4.vcl

```diff
@@ -224,7 +224,48 @@ sub ez_user_context_hash {
     }
 }
 
+// Sub-routine to get purge token.
+sub ez_invalidate_token {
+
+    // Prevent tampering attacks on the token mechanisms
+    if (req.restarts == 0
+        && (req.http.accept ~ "application/vnd.ezplatform.invalidate-token"
+            || req.http.x-backend-purge-token
+        )
+    ) {
+        return (synth(400));
+    }
+
+    if (req.restarts == 0 && req.method == "PURGE") {
+        set req.http.accept = "application/vnd.ezplatform.invalidate-token";
+
+        set req.url = "/http-cache/purgetoken";
```

Comment:
could perhaps be something ala `/_ez_http_purgetoken` to be more like fos approach.

Reply:
ok, so going to use invalidate everywhere ( not a mix of purge and invalidate )..
So, let's use `_ez_http_invalidatetoken`